app/scripts/modules/titus/src/serverGroup/details/rollback/rollbackServerGroup.controller.js

```diff
@@ -66,11 +66,14 @@ module(TITUS_SERVERGROUP_DETAILS_ROLLBACK_ROLLBACKSERVERGROUP_CONTROLLER, [SERVE
       $scope.command = {
         rollbackType: rollbackType,
         rollbackContext: {
+          imageId: previousServerGroup ? previousServerGroup.imageId : undefined,
           rollbackServerGroupName: serverGroup.name,
           restoreServerGroupName: previousServerGroup ? previousServerGroup.name : undefined,
           targetHealthyRollbackPercentage: healthyPercent,
           delayBeforeDisableSeconds: 0,
         },
+        targetGroups: serverGroup.targetGroups,
```

Comment:
My guess is that Titus was never updated when ALBs were introduced, so all titus prev-ami rollbacks just dropped the target groups.